azurerm/internal/services/hpccache/hpc_cache_resource.go

```diff
@@ -86,6 +86,38 @@ func resourceHPCCache() *schema.Resource {
 				ValidateFunc: validation.IntBetween(576, 1500),
 			},
 
+			"ntp_server": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "time.windows.com",
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+
+			"dns_setting": {
```

Comment:
i think setting is redundant here?
```suggestion
			"dns": {
```